step-plans/step-plans-parser/src/main/java/step/repositories/parser/ParsingContext.java

```diff
@@ -100,6 +104,9 @@ protected StackEntry pop() {
 	
 	public AbstractArtefact popCurrentArtefact() {

```

Comment:
To be fully consistant we should rename this popNonWrappingArtifact too or add the information as javadoc...